README.md

```diff
@@ -137,12 +137,14 @@ Models                                                                      | Ca
 [SqueezeNet](https://arxiv.org/pdf/1602.07360)                              |   √   |   ×   |     ×      |
 [DenseNet](https://arxiv.org/abs/1608.06993)                                |   √   |   ×   |     ×      |
 [AllCNN](https://arxiv.org/abs/1412.6806)                                   |   √   |   ×   |     ×      |
-[AlexNet](http://papers.nips.cc/paper/4824-imagenet-classification-with-deep-convolutional-neural-networks)                                 |   √   |   ×   |     ×      |
+[AlexNet](http://papers.nips.cc/paper/4824-imagenet-classification-with-deep-convolutional-neural-networks)                                 |   √   |   √   |     ×      |
```

Comment:
The Tensorflow export also seems to be working could you checkmark that as well.

Reply:
I have a doubt. To recreate AlexNet in Tensorflow, which API should be used so that the saved model will be supported by Fabrik ? I tried using a low level API to create the simple model but I've since come to the conclusion that low level API is not currently supported.